cwatm/hydrological_modules/water_quality/erosed.py

```diff
@@ -39,15 +40,107 @@ class waterquality_erosed(object):
     def __init__(self, model):
         self.var = model.var
         self.model = model
+        self.waterquality_vars = waterquality_vars(model)
 
     def initial(self):
         """
                 INITIAL PART OF THE EROSED MODULE
+
+        Sediment yield per grid cell is calculated with the Modified Universal Soil Loss Equation (MUSLE)
+        Williams (1995)
         """
-        print('erosed init')
+        self.waterquality_vars.initial()
```

Comment:
Line 52: self.waterquality_vars.initial() is already called in routing_kinnematic. no need to call it again. Please test with out this line as well,

Reply:
It worked without the line, I have removed it.